### PR TITLE
Merge pull request #30 from FRC-1721/expand-structure

### PR DESCRIPTION
**Pull request by KenwoodFox**
Branch contains two major features, name is misleading.
- Move all `hungry_toaster` packages to a folder named `hungry_toaster` to differentiate robots.
	- Rename any package made for hungry toaster to the naming convention `robot_name_package_name_here`
 - Create a new folder named `unnamed_toaster` for this years robot

And
 - Start the code for a Networktables to rosmsgs converter

### CODE REVIEW

File: New folder/hungry_toaster_limelight/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+hungry_toaster_limelight(VERSION 2.8.3)
```
Comment (mikeferguson) [inline New folder/hungry_toaster_limelight/CMakeLists.txt]: this isn't right

File: unnamed_toaster/unnamed_toaster_nt_interface/src/nt_interface.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    port_encoder = rospy.Publisher('port_encoder', Int32, queue_size=10)
+    # Init node
+    rospy.init_node('nt_interface', anonymous=True)
+    rate = rospy.Rate(10) # In hz
+
+    while not rospy.is_shutdown():
+        nt_value_starboard_encoder = "0.0"
+        nt_value_port_encoder = "0.0"
+        rospy.loginfo("Got values from NT, current index is: " + str(0))
+        starboard_encoder.publish(nt_value_starboard_encoder)
```
Comment (mikeferguson) [inline unnamed_toaster/unnamed_toaster_nt_interface/src/nt_interface.py:18]: I think you need to publish an Int32 here (and set the .data equal to your values above)

**Review by mikeferguson — APPROVED** [review]

File: New folder/hungry_toaster_limelight/CMakeLists.txt (hunk 1 above)
Comment (KenwoodFox) [reply New folder/hungry_toaster_limelight/CMakeLists.txt]: Where did this file go?

File: unnamed_toaster/unnamed_toaster_nt_interface/src/nt_interface.py (hunk 2 above)
Comment (KenwoodFox) [reply unnamed_toaster/unnamed_toaster_nt_interface/src/nt_interface.py:18]: not sure how to implement the `.data` bit,

**Pull request merged**